glue/viewers/image/state.py

```diff
@@ -449,11 +509,16 @@ def slice_to_bound(slc, size):
         # We now get the fixed resolution buffer
 
         if isinstance(self.layer, BaseData):
+            affine_pretransform = self.viewer_state._affine_pretransform
+            if affine_pretransform is not None:
+                affine_pretransform = affine_pretransform.inverse
             image = self.layer.compute_fixed_resolution_buffer(full_view, target_data=self.viewer_state.reference_data,
-                                                               target_cid=self.attribute, broadcast=False, cache_id=self.uuid)
+                                                               target_cid=self.attribute, broadcast=False, cache_id=self.uuid,
+                                                               affine_transform=self.viewer_state._affine_pretransform.inverse)
```

Comment:
```suggestion
                                                               affine_transform=affine_pretransform)
```
stupid GH blocks suggestions over deleted lines...